src/pages/index.js

```diff
@@ -33,60 +47,156 @@ const cardSection = new Section({
   }
 }, ".cards");
 const popupWithImage = new PopupWithImage('.modal_type_image');
-const api = new Api({
-  baseUrl: "https://around-api.en.tripleten-services.com/v1",
-  headers: {
-    authorization: "32400e4e-bdb6-4554-9f2b-339b0c55f4e6",
-    "Content-Type": "application/json"
-  }
-});
 
 
-// define functions
-function renderProfileDetails(){
-  const { name, description } = userInfo.getUserInfo();
-  nameInput.value = name;
-  jobInput.value = description;
+/* ------------------------------------------------------ */
+/*                    define functions                    */
+/* ------------------------------------------------------ */
+
+function renderProfileDetails(modal){
+  const { name, about } = userInfo.getUserInfo();
+  modal.setInputValues({name, about})
 };
 
 function handleCardAdd(data){
-  const { title, url } = data
-  const cardElement = createCard({title, url})
-  cardSection.addItem(cardElement)
-  cardModal.close()
-};
+  const {name, link} = data;
+  function makeRequest(){
+    return api.addCard(name, link).then((data)=>{
+      cardSection.addItem(createCard(data))
```

Comment:
* You should **clear the inputs** after a successful adding of a new card to let the user add the 2nd card again without having to remove the old data manually. 




```js

    formValidators['cardForm'].disableButton();
    formValidators['cardForm'].resetInputs();
```